plugins/http/client.go

```diff
@@ -116,5 +116,6 @@ func (t *transport) RoundTrip(req *http.Request) (res *http.Response, err error)
 	if res.StatusCode >= http.StatusBadRequest {
 		span.Error(time.Now(), "Errors on handling client")
 	}
+	span.End()
```

Comment:
![image](https://user-images.githubusercontent.com/26432832/96532743-17a9f580-12bf-11eb-9b6e-1d5cc6fe3b59.png)

Hi @Matrix-K I don't see what you mean. In my opinion, this creates new bugs

Reply:
Sorry, I forgot to close the span when roundTrip error.   I'll solve this later.

What I mean is that when the http.Client connection pool is used globally, defer span.End() cannot be closed normally

Just like Image below

![image](https://user-images.githubusercontent.com/19209052/96538962-6d38cf00-12cc-11eb-8b67-892fd4de855b.png)

Reply:
@Matrix-K Can you provide me with a sample project? I think your modified code has the same effect as using `defer` :)